### ISSUE TITLE
loss=0.44 on the SROIE dataset

### ISSUE THREAD

**Issue by ghost** [opconty/pytorch_ctpn#22]
@opconty
I have been training for 1 day on the [SROIE dataset.](https://rrc.cvc.uab.es/?ch=13), the loss is still 0.44 ! 
It works well on other datasets, but not the SROIE dataset?
Am I doing something wrong?
[dataset download link](https://github.com/dhiraa/text-information-extraction#dataset)

![X00016469620](https://user-images.githubusercontent.com/49877135/64128638-27ffc800-cdc0-11e9-9bac-f36a21859996.jpg)
![X00016469622](https://user-images.githubusercontent.com/49877135/64128653-3817a780-cdc0-11e9-9de9-460bfd8f33d8.jpg)

**Comment by opconty** [comment]
Hi,
Could you share your an annotation example?just send me by email: gao.gzhou#gmail.com.

Or you can try my new text detection methods: 
[std](https://github.com/opconty/keras_std)
[std++](https://github.com/opconty/keras_std_plus_plus)

I think std will work better in this task.

Best

**Comment by ghost** [comment]
@opconty 
[sample.zip](https://github.com/opconty/pytorch_ctpn/files/3570491/sample.zip)
or you can download the [entire dataset](https://drive.google.com/drive/folders/1ShItNWXyiY1tFDM5W02bceHuJjyeeJl2)

Why stuck at 0.44 loss?
What are the limitations of pytorch_ctpn?

**Comment by opconty** [comment]
ok, I'll check that when I have time

**Comment by ghost** [comment]
to be continued in courao/ocr.pytorch#6
